docs/getting_started/hopsworksai/guides/getting_started_azure.rst

```diff
@@ -173,7 +173,86 @@ Congratulations, you have successfully connected you Azure account to Hopsworks.
     :align: center
     :figclass: align-center
 
-Step 2: Deploying a Hopsworks cluster
+Step 2: Creating and configuring a storage
+------------------------------------------
+The hopsworks clusters deployed by hopsworks.ai store their data in a container in your Azure accoutn.
```

Comment:
Hopsworks not hopsworks
also account not accoutn
Can you add a spellchecker to your editor, e.g. https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker#:~:text=Open%20up%20VS%20Code%20and,and%20reload%20window%20to%20enable.